### PR TITLE
etcdctlv3: make member list beautiful

### PR DESCRIPTION
**Pull request by xiang90**
```
+------------------+---------+--------+------------------------+------------------------+-----------+
|        ID        | STATUS  |  NAME  |       PEER ADDRS       |      CLIENT ADDRS      | IS LEADER |
+------------------+---------+--------+------------------------+------------------------+-----------+
| 8211f1d0f64f3269 | started | infra1 | http://127.0.0.1:12380 | http://127.0.0.1:12379 | false     |
| 91bc3c398fb3c146 | started | infra2 | http://127.0.0.1:22380 | http://127.0.0.1:22379 | false     |
| fd422379fda50e48 | started | infra3 | http://127.0.0.1:32380 | http://127.0.0.1:32379 | true      |
+------------------+---------+--------+------------------------+------------------------+-----------+
```

### CODE REVIEW

**Comment by heyitsanthony** [comment]
Can this be done through the printer interface so `-w` will work?

**Comment by xiang90** [comment]
@heyitsanthony Hrm... we want a protobuf format for this?

**Comment by xiang90** [comment]
@heyitsanthony I will make json work.

**Comment by xiang90** [comment]
@heyitsanthony PTAL

File: etcdctlv3/command/member_command.go [hunk 1]
```diff
@@ -166,12 +168,33 @@ func memberListCommandFunc(cmd *cobra.Command, args []string) {
 		ExitWithError(ExitError, err)
 	}
 
-	// use https://github.com/olekukonko/tablewriter to print out a pretty table?
-	for _, m := range resp.Members {
-		if len(m.Name) == 0 {
-			fmt.Printf("%16x[unstarted]: peerURLs=%s\n", m.ID, strings.Join(m.PeerURLs, ","))
-		} else {
-			fmt.Printf("%16x: name=%s peerURLs=%s clientURLs=%s isLeader=%v\n", m.ID, m.Name, strings.Join(m.PeerURLs, ","), strings.Join(m.ClientURLs, ","), m.IsLeader)
+	outputType, _ := cmd.Flags().GetString("write-out")
```
Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: add `MemberList(*MemberListResponse)` to `printer` instead of having to specially handle the flag?

Comment (xiang90) [inline etcdctlv3/command/member_command.go]: now the printer is just for kv responses. so we want to make it more generic?

Comment (xiang90) [inline etcdctlv3/command/member_command.go]: i do not feel it is necessary to make every utility commands to accept -w. member list might be an exception when people want to programmatically parse the result.

Comment (heyitsanthony) [inline etcdctlv3/command/member_command.go]: If its purpose is to print some kind of `*Response` it seems like it should use `printer` / support `-w`. Commands like mirror maker or defrag wouldn't need to bother with `-w`.

Comment (xiang90) [inline etcdctlv3/command/member_command.go]: ok. make sense.

**Comment by xiang90** [comment]
PTAL

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**